src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -833,6 +837,14 @@ subroutine Restart(this, bounds, ncid, flag)
          dim1name='pft', dim2name='levcan', switchdim=.true., &
          long_name='Maximum carboxylation rate at 25 celcius for canopy layers', units='umol CO2/m**2/s', &
          interpinic_flag='interp', readvar=readvar, data=this%jmx25_z_patch)
+      call restartvar(ncid=ncid, flag=flag, varname='vcmx_prevyr', xtype=ncd_double,  &
+         dim1name='pft', dim2name='levcan', switchdim=.true., &
+         long_name='avg carboxylation rate at 25 celsius for canopy layers', units='umol CO2/m**2/s', &
+         interpinic_flag='interp', readvar=readvar, data=this%vcmx_prevyr)
+      call restartvar(ncid=ncid, flag=flag, varname='jmx_prevyr', xtype=ncd_double,  &
+         dim1name='pft', dim2name='levcan', switchdim=.true., &
+         long_name='avg carboxylation rate at 25 celsius for canopy layers', units='umol CO2/m**2/s', &
+         interpinic_flag='interp', readvar=readvar, data=this%jmx_prevyr)
```

Comment:
long names here should match what we're using in the description above, something about the previous year?  

Minor correction, but `jmx25_z_patch` and  `jmx_prevyr` long_name should be maximum rate of electron transport at 25 celsius for canopy layers, with units `umol electron/m**2/s`

Reply:
We'll fix this in #1060